system/indy-node-tests/TestCatchUpSuite.py

```diff
@@ -126,12 +130,14 @@ class TestCatchUpSuite:
             pool_handler, wallet_handler, trustee_did, nyms_count=nyms_count, timeout=WRITE_READ_TIMEOUT
         )
 
-        await ensure_pool_is_functional(pool_handler, wallet_handler, trustee_did, nyms_count=100)
+        await ensure_pool_is_functional(
+            pool_handler, wallet_handler, trustee_did, nyms_count=200, timeout=WRITE_READ_TIMEOUT
+        )
         await ensure_pool_is_in_sync(nodes_num=nodes_num)
         await ensure_state_root_hashes_are_in_sync(pool_handler, wallet_handler, trustee_did)
 
     @pytest.mark.parametrize('check_reachability', [False, True])
-    @pytest.mark.parametrize('nyms_count', [1, 25, 50])
```

Comment:
I guess 50 was removed due to being almost duplicate of test case with 25 NYMs, did I get it right?